CondCore/CondDB/src/CoralServiceFactory.cc

```diff
@@ -20,14 +20,6 @@ cond::CoralServiceFactory::get() {
 
 coral::Service*
 cond::CoralServiceFactory::create(const std::string& componentname) const {
- coral::Service* sp=CoralServicePluginFactory::get()->create(componentname,componentname);
- if(sp==nullptr) {
-   throw cond::Exception("CoralServiceFactory")
-     << "CoralServiceFactory:\n"
-     << "Cannot find coral service: "
-     << componentname << "\n"
-     << "Perhaps the name is misspelled or is not a Plugin?\n"
-     << "Try running EdmPluginDump to obtain a list of available Plugins.";
```

Comment:
This exception was never thrown (since `PluginFactory::create()` throws for non-existing plugin), so I removed it.